JobConfig/ensemble/python/normalizations.py

```diff
@@ -131,7 +146,7 @@ def cry_offspill_normalization(livetime, run_mode = '1BB'):
     if(run_mode == '2BB'):
       # 2BB
       offspill_dutyfactor = 0.246
```

Comment:
here too we need a central location for common numbers. Analyzers will need to know these as well. It should become an issue.

Reply:
yes agreed

Reply:
https://github.com/Mu2e/Production/issues/334